scripts/lst-sensitivity.py

```diff
@@ -67,59 +70,64 @@
         continue
     tab[key].format = '8f'
 
-print(tab)
-emed = np.sqrt(E[1:] * E[:-1])
 
-plt.plot(emed[:-1], tab['hadron_rate'], label='Hadron rate', marker='o')
-plt.plot(emed[:-1], tab['gamma_rate'], label='Gamma rate', marker='o')
+#tab.to_csv("table.txt")
+
+plot_utils.sensitivity_plot_comparison(eb, E, best_sens)
+plt.show()
+
+egeom = np.sqrt(E[1:] * E[:-1])
+
+plt.plot(egeom[:-1], tab['hadron_rate'], label = 'Hadron rate', marker = 'o')
+plt.plot(egeom[:-1], tab['gamma_rate'], label = 'Gamma rate', marker = 'o')
 plt.legend()
 plt.xscale('log')
 plt.xlabel('Energy (GeV)')
 plt.ylabel('events / min')
-plt.show()
+#plt.show()
```

Comment:
tbr

Reply:
we should write these images to files, I'll leave this uncomment for the moment.